redisson/src/main/java/org/redisson/eviction/EvictionTask.java

```diff
@@ -33,7 +33,7 @@
  */
 abstract class EvictionTask implements TimerTask {
 
-    private final Logger log = LoggerFactory.getLogger(getClass());
+    private static final Logger log = LoggerFactory.getLogger(EvictionTask.class);
```

Comment:
this needs to remain unchanged or be added to all inheritance classes with concrete class, not EvictionTask.